Yoast/wordpress-seo
# Remove headingId from howto attributes

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* This attribute was unused, and can be removed.

## Test instructions

This PR can be tested by following these steps:

* Make a How-To block in a post while on trunk (and publish).
* Switch to this branch and build everything.
* Go back to that post, and verify that everything still works in the editor (editting capabilities, sidebar, no 'this block has been edited' warning from Gutenberg) and in the frontend (json-ld, title).

## Quality assurance
